src/server/strategies/bearer.strategy.js

```diff
@@ -0,0 +1,32 @@
+const Strategy = require('passport-http-bearer').Strategy;
+const request = require('superagent');
+
+/**
+ * Bearer Strategy
+ * Requires ENV variables for introspecting the token
+ */
+module.exports.strategy = new Strategy(
+	function(token, done) {
+
+		if (!process.env.introspectionUrl) {
+			return done(new Error('Invalid introspection endpoint.'));
+		}
+
+		request
+		.post(process.env.introspectionUrl)
+		.set('content-type', 'application/x-www-form-urlencoded')
+		.send({token: token, client_id: process.env.clientId, client_secret: process.env.clientSecret})
```

Comment:
Is it customary to send the token along with the client_id and client_secret used to generate it?  Seems you should keep the secret to yourself?

Reply:
yes its required